src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -57,6 +59,20 @@ public void start() {
         gamerules.get(GameRules.DO_IMMEDIATE_RESPAWN).set(true, server);
         gamerules.get(GameRules.DO_ENTITY_DROPS).set(false, server);
 
+        final var worldBorder = server.getOverworld().getWorldBorder();
+        worldBorder.setSize(Molehunt.CONFIG.getInitialWorldSize());
+        if (Molehunt.CONFIG.getBorderShrinkingStartingTimeOffset() < Molehunt.CONFIG.getGameDuration()) {
+            borderTimeOffsetTime.schedule(new TimerTask() {
+                @Override
+                public void run() {
+                    worldBorder.interpolateSize(
+                            Molehunt.CONFIG.getInitialWorldSize(),
+                            Molehunt.CONFIG.getFinalWorldSize(),
+                            (long) (Molehunt.CONFIG.getGameDuration() - Molehunt.CONFIG.getBorderShrinkingStartingTimeOffset()) * 60 * 60);

```

Comment:
60*60 is for hour not minutes, isn't it?

Reply:
yeah I didn't think this through